arch/risc-v/include/bl808/irq.h

```diff
@@ -31,10 +31,18 @@
 
 /* Map RISC-V exception code to NuttX IRQ */
 
-#define NR_IRQS (RISCV_IRQ_SEXT + 57)
+#define NR_IRQS (RISCV_IRQ_SEXT + 114)
 
 #define BL808_IRQ_NUM_BASE (16)
+#define BL808_M0_IRQ_OFFSET (64) // IRQs tied to M0 core are given a virtual IRQ number
```

Comment:
I think we need this to satisfy [`nxstyle`](https://lupyuen.github.io/articles/pr#check-coding-style):

```suggestion
#define BL808_M0_IRQ_OFFSET (64) /* IRQs tied to M0 core are given a virtual IRQ number */
```